packages/toolpad-app/src/runtime/evalJsBindings.ts

```diff
@@ -171,12 +171,12 @@ export default function evalJsBindings(
     if (expression) {
       const computed = computationStatuses.get(expression);
       if (computed) {
-        if (computed.result) {
+        if (!computed.result) {
```

Comment:
The reason why some charts were showing errors before loading was because they were using a cached result, but using this result didn't let them recognize that the query was loading, unlike in the charts with uncached results here...
Here i am not caching if there is an error. Maybe there is a better solution?